src/errno.rs

```diff
@@ -25,6 +25,10 @@ cfg_if! {
         unsafe fn errno_location() -> *mut c_int {
             libc::__errno_location()
         }
+    } else if #[cfg(any(target_os = "illumos", target_os = "solaris"))] {
```

Comment:
It's hard to see in the browser, but on illumos and Solaris, it's three underscores, but only two on android/netbsd/openbsd, so I don't believe it can (unless I'm misunderstanding how you wanted to combine them).

Reply:
You're right; that's awfully hard to see.